client/login/wp-login/index.jsx

```diff
@@ -79,6 +79,9 @@ export class Login extends React.Component {
 
 					<DocumentHead title={ translate( 'Log In', { textOnly: true } ) } />
 
+					<DocumentHead meta={ [ { name: 'robots', content: 'noindex, follow' } ] } />
```

Comment:
If we decide to keep it, the `meta` should be merged into the `DocumentHead` element above.